xmr-btc/src/lib.rs

```diff
@@ -60,4 +60,26 @@ pub mod monero;
 pub mod serde;
 pub mod transport;
 
+use crate::bitcoin::{BlockHeight, TransactionBlockHeight, WatchForRawTransaction};
 pub use cross_curve_dleq;
+
+pub async fn current_epoch<W>(
```

Comment:
This should go in `src/bitcoin.rs`.

Please keep in mind that `lib.rs` should and will be deleted.